concepticondata/conceptlists/Reinhard-1970-337.tsv

```diff
@@ -0,0 +1,338 @@
+Kusunda-1972-337-40	40	sleep (1st)	1585	SLEEP
+Kusunda-1972-337-41	41	squeeze (1st)	1414	SQUEEZE
+Kusunda-1972-337-42	42	hold (1st)	1448	HOLD
+Kusunda-1972-337-43	43	bring (1st)	692	BRING
+Kusunda-1972-337-44	44	hold fast (1st)		
+Kusunda-1972-337-45	45	tie (1st)	1917	TIE
+Kusunda-1972-337-46	46	know (1st)	1410	KNOW (SOMETHING)
+Kusunda-1972-337-47	47	at	1461	AT
+Kusunda-1972-337-48	48	suck (1st)	1421	SUCK
+Kusunda-1972-337-49	49	drink (1st)	1401	DRINK
+Kusunda-1972-337-50	50	round	1395	ROUND
+Kusunda-1972-337-51	51	big	1202	BIG
+Kusunda-1972-337-52	52	get (1st)	694	GET
+Kusunda-1972-337-53	53	tuber (large) (Dioscorea daemonia)		
+Kusunda-1972-337-54	54	to, in	1460	IN
+Kusunda-1972-337-55	55	three	492	THREE
+Kusunda-1972-337-56	56	chicken	1318	CHICKEN
+Kusunda-1972-337-57	57	large river	666	RIVER
+Kusunda-1972-337-58	58	smoke	778	SMOKE (EXHAUST)
+Kusunda-1972-337-59	59	hill	614	HILL
+Kusunda-1972-337-60	60	rakshi		
+Kusunda-1972-337-61	61	stand (1st)	1442	STAND
+Kusunda-1972-337-62	62	drum (two ended type)	908	DRUM
+Kusunda-1972-337-63	63	salah tree		
+Kusunda-1972-337-64	64	son	1620	SON
+Kusunda-1972-337-65	65	cloud	1489	CLOUD
+Kusunda-1972-337-66	66	dust	2	DUST
+Kusunda-1972-337-67	67	push (1st)	1452	PUSH
+Kusunda-1972-337-68	68	husband	1200	HUSBAND
+Kusunda-1972-337-69	69	one week	1469	WEEK
+Kusunda-1972-337-70	70	sing (1st)	1261	SING
+Kusunda-1972-337-71	71	fog	249	FOG
+Kusunda-1972-337-72	72	grease	323	FAT (ORGANIC SUBSTANCE)
+Kusunda-1972-337-73	73	grass	606	GRASS
+Kusunda-1972-337-74	74	in	1460	IN
+Kusunda-1972-337-75	75	heart	1223	HEART
+Kusunda-1972-337-76	76	sand	671	SAND
+Kusunda-1972-337-77	77	game of stones		
+Kusunda-1972-337-78	78	sharp	1396	SHARP
+Kusunda-1972-337-79	79	forest chicken		
+Kusunda-1972-337-80	80	play (1st)	1413	PLAY
+Kusunda-1972-337-81	81	move (1st)	2874	MOVE (ONESELF)
+Kusunda-1972-337-82	82	wife's father		
+Kusunda-1972-337-83	83	flower	239	FLOWER
+Kusunda-1972-337-84	84	inside	1606	INSIDE
+Kusunda-1972-337-85	85	language	1307	LANGUAGE
+Kusunda-1972-337-86	86	meat	634	MEAT
+Kusunda-1972-337-87	87	breathe (1st)	1407	BREATHE
+Kusunda-1972-337-88	88	male sexorgan		
+Kusunda-1972-337-89	89	he/ person	1211	HE
+Kusunda-1972-337-90	90	wood 	2141	TREE OR WOOD
+Kusunda-1972-337-91	91	female sex organ		
+Kusunda-1972-337-92	92	wife's mother		
+Kusunda-1972-337-93	93	a kind of tree		
+Kusunda-1972-337-94	94	thick	1244	THICK
+Kusunda-1972-337-95	95	One-sided drum	908	DRUM
+Kusunda-1972-337-96	96	forest	420	FOREST
+Kusunda-1972-337-97	97	he, she	2642	HE OR SHE
+Kusunda-1972-337-98	98	they	817	THEY
+Kusunda-1972-337-99	99	seed	714	SEED
+Kusunda-1972-337-100	100	sweet/ sap	717	SWEET
+Kusunda-1972-337-101	101	skin	763	SKIN
+Kusunda-1972-337-102	102	feather	1201	FEATHER
+Kusunda-1972-337-103	103	egg	744	EGG
+Kusunda-1972-337-104	104	unpleasant	26	UNPLEASANT
+Kusunda-1972-337-105	105	this morning	1339	MORNING
+Kusunda-1972-337-106	106	tomorrow	1329	TOMORROW
+Kusunda-1972-337-107	107	mother's older brother		
```

Comment:
there's a kinship term for this, right? please check for "uncle", and add, if you don't find "uncle (older brother of mother)" or similar, we should have similar examples in concepticon already